### ISSUE TITLE
renderHTML does not include HTML of custom blocks

### ISSUE THREAD

**Issue by gopeter** [verbb/vizy#108]
**Description**

I've added two custom blocks ("2 Columns" and "3 Columns") which holds nested Vizy fields. If I want to query them with Graphql, they aren't included in the `renderHTML` response. Iterating through `nodes` and trying to get the `html` from there, I just get an empty string (but I can't provide an example yet because of #107 😄 )

**Additional info**
- Plugin version: 1.0.10
- Craft version: Craft Pro 3.7.33
- Multi-site: No

![image](https://user-images.githubusercontent.com/1242960/155998908-5d689522-a464-4ebe-9c98-ec33a0a211dc.png)

![image](https://user-images.githubusercontent.com/1242960/155999058-4c3fae3a-c15e-4650-a402-b4ea1e7e9597.png)

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by gopeter** [comment]
Works fine, thanks!

**Comment by engram-design** [comment]
Fixed in [1.0.11](https://github.com/verbb/vizy/releases/tag/1.0.11)
